modular_skyrat/modules/conveyor_sorter/code/conveyor_sorter.dm

```diff
@@ -7,6 +7,9 @@
 	var/list/spawned_sorters = list()
 	///the list of things that are currently within the sorting list
 	var/list/current_sort = list()
+	var/max_sorters = 4 //This controls the maximum amount of sorters that can be spawned by one lister item.
+	var/max_items = 5 //This controls the maximum amount of items that can be added to the sorting list.
+	var/conveyor_type = /obj/effect/decal/cleanable/conveyor_sorter // This is used for the improved sorter, so that it can use the improved sorter type instead of the normal sorter type.
```

Comment:
Autodoc requires it to be on the line before, and using `///` instead of just `//`.
```suggestion
	/// This controls the maximum amount of sorters that can be spawned by one lister item.
	var/max_sorters = 4
	/// This controls the maximum amount of items that can be added to the sorting list.
	var/max_items = 5
	/// This is used for the improved sorter, so that it can use the improved sorter type instead of the normal sorter type.
	var/conveyor_type = /obj/effect/decal/cleanable/conveyor_sorter
```